pkg/bootstrap/bootstrap_config.go

```diff
@@ -74,19 +73,18 @@ var (
 	requiredEnvoyStatsMatcherInclusionPrefixes = "cluster_manager,listener_manager,http_mixer_filter,tcp_mixer_filter,server,cluster.xds-grpc"
 	requiredEnvoyStatsMatcherInclusionSuffix   = "ssl_context_update_by_sds"
 
-	metadataExchangeKeys = strings.Join(
-		[]string{
-			model.NodeMetadataInstanceName,
-			model.NodeMetadataNamespace,
-			model.NodeMetadataInstanceIPs,
-			model.NodeMetadataLabels,
-			model.NodeMetadataOwner,
-			model.NodeMetadataPlatformMetadata,
-			model.NodeMetadataWorkloadName,
-			model.NodeMetadataCanonicalTelemetryService,
-			model.NodeMetadataMeshID,
-			model.NodeMetadataServiceAccount,
-		}, ",")
+	metadataExchangeKeys = []string{
```

Comment:
do we care about keeping this in-sync with the `json` serializations from the struct?

Reply:
Good point. So as far as I can tell we cannot have a single const that declares both here and the json serialization (cant do something like ```name string fmt.Sprintf(`json:"%s"`, SomeConstant)```), maybe just a strongly worded comment? Open to suggestions here, this was very disappointing when I realized this

Reply:
I think a strongly-worded comment and some unit test are probably the best we can do.